DESCRIPTION.md

```diff
@@ -10,8 +10,10 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 - v3.0.2(Unreleased)
 
-  - Fixed a memory leak in the logging module of the Cython extension.
+  - Fixed a memory leak in the logging module of the Cython extension
   - Fixed a bug where the `put` command on AWS raised `AttributeError` when the file size was larger than 200M.
+  - Validate SSO URL before accessing
```

Comment:
Where are we applying `is_valid_url` to validate the URL? Is that in a separate PR? 
Let's rephrase as: `"Added SSO URL validation before opening it in browser in external browser authentication"` and `"Added`url_util` module to provide URL validation utilities. "`?